README.md

```diff
@@ -32,4 +32,71 @@
 * Видео, прикрепленное к PR, обязательно должно показывать, что все koans пройдены.
 
 ### Дедлайн
-2018-07-09 18:00:00 UTC+3
+2019-07-09 18:00:00 UTC+3
```

Comment:
Do not change this file! (I know, why you did it, but please, don't do it)

Reply:
I needed to change those files so now they are the same as updated in summer-2019